docs/kubeapps-on-bkpr.md

```diff
@@ -0,0 +1,122 @@
+# Installing Kubeapps on BKPR
+
+This guide documents the installation of [Kubeapps](https://kubeapps.com/) to your Kubernetes Production Runtime (BKPR) cluster. You will first install the [Helm](https://www.helm.sh/) package manager to the cluster, followed by the installation of Kubeapps using the [Kubeapps Helm chart](https://hub.kubeapps.com/charts/bitnami/kubeapps).
+
+![Kubeapps Application Catalog](images/kubeapps-app-catalog.png)
+
+The guide assumes you have a Kubernetes cluster with BKPR already installed. The [BKPR installation guide](install.md) documents the process of installing the BKPR client, followed by installing the BKPR server side components to the cluster.
+
+## Step 1: Install the Helm client
+
+Follow the instructions in [Helm installation guide](https://docs.helm.sh/using_helm/#installing-the-helm-client) to install the latest release of the Helm binary to your machine.
+
+## Step 2: Install the Tiller server
+
+Tiller is the server-side component of Helm that runs inside your Kubernetes cluster. To install the Tiller server to the cluster you should use the `helm` client installed by the previous step. However, before we do so, for RBAC enabled clusters we need to create a service account for the Tiller server.
```

Comment:
"To install the Tiller server to the cluster you should use the `helm` client installed by the previous step" -> The previous step has a link to Helm instructions and this paragraph doesn't mention any commands. How would a user know how to install Tiller? We should either add Tiller-specific commands here or link to a doc for Tiller setup or mention that the previous link also covers Tiller setup (if it does).